.pre-commit-config.yaml.jinja

```diff
@@ -68,6 +70,14 @@ repos:
       - id: check-xml
       - id: mixed-line-ending
         args: ["--fix=lf"]
+  - repo: https://github.com/oca/maintainer-tools
+    rev: 2c66e72 # Leave as is for stability
+    hooks:
+      # update the NOT INSTALLABLE ADDONS section above
```

Comment:
I think this hook should be the 1st or 2nd one, to make all other hooks skip this code.